### ISSUE TITLE
Your TF implementation vs others (comparison/ differentiation)

### ISSUE THREAD

**Issue by Nikolai10** [sayakpaul/swin-transformers-tf#5]
Hello @sayakpaul, 

thank you for your great work. While searching for a TensorFlow-based Swin Transformer implementation, I came across:

- your repo through the [official implementation website](https://github.com/microsoft/Swin-Transformer),
- https://github.com/rishigami/Swin-Transformer-TF and
- https://github.com/VcampSoldiers/Swin-Transformer-Tensorflow.

I would be quite thankful, if you could elaborate on the differences between these variants. I assume they all "proved" functional correctness to some degree. From a scientific point of view, what sets your work apart from others?

Thanks for the clarification, 

Nikolai

**Comment by sayakpaul** [comment]
I possibly cannot go into those individual repositories and tell you what differentiates mine. I am sorry. 

For functional correctness, I have validated all the released checkpoints on the ImageNet-1k validation set ensuring they reach the expected numbers (with some tolerance). I have demonstrated their transfer learning abilities as well. 

If they above-mentioned repositories provide all of that then good otherwise I would be worried.
